tests/integration/test_vulnerability_detector/test_feeds/data/test_cases/cases_validate_json_feed_content.yaml

```diff
@@ -39,6 +39,19 @@
     decompressed_file: /tmp/alas.json
     url: https://feed.wazuh.com/vulnerability-detector/ALAS/1/alas.json.gz
 
+
+- name: Amazon Linux 2023
+  description: JSON Amazon Linux provider
+  configuration_parameters:
+  metadata:
+    provider_name: Amazon Linux 2023
+    expected_format: application/gzip
+    path: /tmp/alas.json.gz
+    extension: gz
+    decompressed_file: /tmp/alas.json
+    url: https://feed.wazuh.com/vulnerability-detector/ALAS/2023/alas2023.json.gz
+
+
```

Comment:
To fix this check:
- https://github.com/wazuh/wazuh-qa/actions/runs/6241022523/job/16942353629?pr=4482

Avoid adding the following line breaks:
```suggestion
- name: Amazon Linux 2023
  description: JSON Amazon Linux provider
  configuration_parameters:
  metadata:
    provider_name: Amazon Linux 2023
    expected_format: application/gzip
    path: /tmp/alas.json.gz
    extension: gz
    decompressed_file: /tmp/alas.json
    url: https://feed.wazuh.com/vulnerability-detector/ALAS/2023/alas2023.json.gz

```